### ISSUE TITLE
Distinguish between hard and soft returns in note-taking popover to improve workflow

### ISSUE THREAD

**Issue by digitalethics** [johnfactotum/foliate#383]
Currently the line length in the note-taking popover automatically prompts a line or word wrap once the text has reached the margin settings of the window. As there is no distinction made between hard and soft returns (and apparently not necessary as there are no formatting options), pressing <kbd>↵ Enter</kbd> and <kbd>⇧ Shift</kbd> + <kbd>↵ Enter</kbd> yield the same results. Pressing <kbd>Esc</kbd> exits the note-taking window. While this workflow is fine, I think it can be improved. 

With <kbd>Esc</kbd> I typically expect a return to the main window (so far so good) without saving any annotations. Would it be possible to save the annotations and exit the note-taking popover by pressing <kbd>↵ Enter</kbd> instead and leave <kbd>Esc</kbd> for exit without saving the notes? Perhaps a soft break via <kbd>⇧ Shift</kbd> + <kbd>↵ Enter</kbd> could then be used for a line break? I'm not sure if this will be more confusing for new users but to me it appears to be more efficient and more intuitive. Maybe there is a better solution to this problem other than distinguishing between hard and soft returns when note-taking?

**Comment by johnfactotum** [comment]
> As there is no distinction made between hard and soft returns (and apparently not necessary as there are no formatting options), pressing ↵ Enter and ⇧ Shift + ↵ Enter yield the same results.

Yes, it's plain text so there's no distinction. To have that you basically need to implement rich text.

> With Esc I typically expect a return to the main window (so far so good) without saving any annotations.

All settings/changes can be expected to be applied/saved immediately unless there's a "Save" button.
